crates/ruma-client-api/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [unreleased]
 
+Improvements:
+
+* Add convenience constructors for enabling lazy-loading in filters 
```

Comment:
Can we do `-` lists from now on? It's easier to distinguish from `*em*` / `**strong**` MD syntax.

Reply:
No problem. I actually realized yesterday tfat we had switched the syntax but I thought it was maybe my fault because I usually use the dash if I don't pay attention.

Reply:
Hah no it was me who started it, intentionally (but w/o telling anybody about it) 😄